proxy/core/src/main/resources/cloudstate-common.conf

```diff
@@ -47,16 +52,17 @@ akka {
   cluster {
     shutdown-after-unsuccessful-join-seed-nodes = 60s
 
+    // FIXME consider setting distributed-data.max-delta-elements = <N>
+    // FIXME consider setting distributed-data.delta-crdt.max-delta-size = <N>
```

Comment:
@pvlugter @patriknw These are here since they are a part of the migration docs, and I didn't want them to be forgotten, but I also don't know what to set them to. Advise?

Reply:
Remove, and use defaults until you see a problem.
Only mentioned in migration guide because the defaults were changed (to better defaults).